### PR TITLE
Updates & TODO's

### PR DESCRIPTION
**Pull request by marbar3778**
Im changing some wording and will be adding sections in, based on the info which is needed.

let me know of what you think as I keep pushing.

### CODE REVIEW

File: src/components/sections/About.js [hunk 1]
```diff
@@ -66,19 +66,22 @@ const About = () => (
               <h2>Be ahead of the curve</h2>
               <p>
                 Earn crypto for supporting validators of blockchain networks.
-                Stake with us to build the future of decentralized finance.
+                Stake with us to build the future we deserve.
```
Comment (foodaka) [inline src/components/sections/About.js]: 👍

File: src/pages/notes.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# TODO add pages for cosmos and kusama
```
Comment (foodaka) [inline src/pages/notes.md]: 👍

Comment (foodaka) [reply src/pages/notes.md]: @marbar3778 when we link to the cosmos and kusama pages, what type of content do you want there?

Comment (marbar3778) [reply src/pages/notes.md]: just some docs maybe

**Comment by marbar3778** [comment]
will merge this into the landing page branch

**Pull request merged**